Enhancement/testnets contracts

409H opened the pull request:
* Added Kovan Testnet support
* Made the UI on the contracts page a little easier if the contract can or cannot be ownership transferred

seaona commented:
## Checked:
- [x] Successfully detecting Kovan network
- [x] Successfully detecting network switching
- [x] Correct page for "Supported Networks"

## Issues:
- **Not detecting deployed contracts on Kovan**: 
Owner address 0x07be9763a718c0539017e2ab6fc42853b4aeeb6b
Contract address: 0x2568e866887137b6984907f290deab905730814a
![image](https://user-images.githubusercontent.com/54408225/159241606-95c08511-eff7-4206-b6a2-ef3bf601ad3b.png)
![image](https://user-images.githubusercontent.com/54408225/159241319-a2d1128d-bc8b-482f-b12d-aabada24c820.png)

- **Not detecting ERC20 tokens on Kovan**: this is a problem that affects other networks, so I've opened [a separated issue](https://github.com/409H/eoa-migrator/issues/8)

409H commented:
Hey @seaona - I have added logic to have custom tokenlists for each network now so it now will find tokens if you change from mainnet to a different network. There is not a maintained kovan tokenlist, so I have added the 2 tokens (`LINK`, `DAI`) into the custom field in [39cba98](https://github.com/409H/eoa-migrator/pull/5/commits/39cba98b25b4ab081dc16127aaff9e43c417b0f5) to help you with testing.

seaona commented:
Thanks!! Working for my side
